pyEpiabm/pyEpiabm/sweep/host_progression_sweep.py

```diff
@@ -66,22 +109,30 @@ def set_infectiousness(person: Person):
         ----------
         Person : Person
             Instance of person class with infection status attributes
+        time : float
+            Current simulation time
 
         Returns
         -------
         float
-            Infectiousness of a person
+            Initial infectiousness of a person
+        float
+            Infection start time of a person
```

Comment:
Does it return these? I can't see it in the function call

Reply:
No indeed, I was confused with the documentation formatting.